app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -70,27 +74,26 @@ src_install() {
 	default
 
 	newinitd "${FILESDIR}"/lircd-0.8.6-r2 lircd
-	newinitd "${FILESDIR}"/lircmd lircmd
+	newinitd "${FILESDIR}"/lircmd-0.9.4a-r2 lircmd
 	newconfd "${FILESDIR}"/lircd.conf.4 lircd
+	newconfd "${FILESDIR}"/lircmd.conf lircmd
 
 	insinto /etc/modprobe.d/
 	newins "${FILESDIR}"/modprobed.lirc lirc.conf
 
-	newinitd "${FILESDIR}"/irexec-initd irexec
+	newinitd "${FILESDIR}"/irexec-initd-0.9.4a-r2 irexec
 	newconfd "${FILESDIR}"/irexec-confd irexec
 
-	if use doc ; then
-		dodoc -r doc/html
-	fi
-
 	keepdir /etc/lirc
 	if [[ -e "${D}"/etc/lirc/lircd.conf ]]; then
 		newdoc "${D}"/etc/lirc/lircd.conf lircd.conf.example
 	fi
 
-	if ! use static-libs; then
-		rm "${D}/usr/$(get_libdir)/liblirc_client.la" || die
-	fi
+	prune_libtool_files --modules
```

Comment:
Close to perfect, you were just missing `--all`, because you definitely want to kill those files.
Any how, https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives documents better practices, i.e., just use `find "${D}" -name '*.la' -delete || die` instead of `prune_libtool_files` from `eutils`.